specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -136,7 +136,7 @@ function ConditionalFormatter({
           <div className="flex items-center gap-2 pr-2">
             <div className="h-full flex-1">
               <Input.Text
-                className="h-full pr-2"
```

Comment:
(optional) just a FYI that adding `fiex` to the parent of this Input would also fix the issue with inconsistent height